WeRadiate-ThermoSense.ino

```diff
@@ -405,6 +464,214 @@ void loop(void)
         {
         // put your main code here, to run repeatedly:
         gCatena.poll();
-        Serial.println("hello world");
-        delay(60000);
+        }
+
+void startSendingUplink(void)
+        {
+        TxBuffer_t b;
+        LedPattern savedLed = gLed.Set(LedPattern::Measuring);
+
+        b.begin();
+        FlagsSensor3 flag;
+
+        flag = FlagsSensor3(0);
+
+        b.put(FormatSensor2); /* the flag for this record format */
+        uint8_t * const pFlag = b.getp();
+        b.put(0x00); /* will be set to the flags */
+
+        // vBat is sent as 5000 * v
+        float vBat = gCatena.ReadVbat();
+        gCatena.SafePrintf("vBat:    %d mV\n", (int) (vBat * 1000.0f));
+        b.putV(vBat);
+        flag |= FlagsSensor3::FlagVbat;
+      
+        // vBus is sent as 5000 * v
+        float vBus = gCatena.ReadVbus();
+        gCatena.SafePrintf("vBus:    %d mV\n", (int) (vBus * 1000.0f));
+        fUsbPower = (vBus > 3.0) ? true : false;
+
+        uint32_t bootCount;
+        if (gCatena.getBootCount(bootCount))
+                {
+                b.putBootCountLsb(bootCount);
+                flag |= FlagsSensor3::FlagBoot;
+                }
+
+        if (fBme)
+                {
+                Adafruit_BME280::Measurements m = gBme.readTemperaturePressureHumidity();
+                // temperature is 2 bytes from -0x80.00 to +0x7F.FF degrees C
+                // pressure is 2 bytes, hPa * 10.
+                // humidity is one byte, where 0 == 0/256 and 0xFF == 255/256.
+                gCatena.SafePrintf(
+                        "BME280:  T: %d P: %d RH: %d\n",
+                        (int) m.Temperature,
+                        (int) m.Pressure,
+                        (int) m.Humidity
+                        );
+                b.putT(m.Temperature);
+                b.putP(m.Pressure);
+                b.putRH(m.Humidity);
+    
+                flag |= FlagsSensor3::FlagTPH;
+                }
+
+        if (fLux)
```

Comment:
I don't think they need lux; so the sensor can be left in stand by and you can delete this whole clause.